mitodl/mitxpro
# style: add requested tweaks in webinar designs

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/hq/issues/2818

#### What's this PR do?
Updates the webinar page design as per the suggestions inhttps://github.com/mitodl/hq/issues/2818

#### How should this be manually tested?
- Create webinars (Both `On-Demand` and `Upcoming`)
- Check the date and time suggestions are met on webinars list page
- Check that the date and time suggestions are met on the webinar details page
- Check that the display is good in both desktop and mobile


#### Screenshots (if appropriate)
**Webinars List** 

**AFTER**
![screencapture-xpro-odl-local-8053-webinars-2023-10-30-16_44_32](https://github.com/mitodl/mitxpro/assets/34372316/d6e3a61e-0383-4b38-b0c3-c0daf6fb6dd7)

**BEFORE**
![screencapture-xpro-odl-local-8053-webinars-2023-10-30-16_55_26](https://github.com/mitodl/mitxpro/assets/34372316/534a50e8-b84f-453e-a630-d5782d15dec7)


--- 

**Webinars Detail Page** 

**AFTER**

![screencapture-xpro-odl-local-8053-webinars-one-demand-webinar-2023-10-30-16_51_47](https://github.com/mitodl/mitxpro/assets/34372316/e9f5df9f-389f-4ffc-b49e-6ae06dcd50c7)

**BEFORE**
![screencapture-xpro-odl-local-8053-webinars-one-demand-webinar-2023-10-30-16_55_14](https://github.com/mitodl/mitxpro/assets/34372316/e409256b-a1d8-496e-94da-1356ba2516a7)
